src/parser/bison_parser.y

```diff
@@ -689,12 +689,12 @@ unary_expr:
 
 binary_expr:
 		comp_expr
-	|	operand '-' operand			{ $$ = Expr::makeOpBinary($1, '-', $3); }
-	|	operand '+' operand			{ $$ = Expr::makeOpBinary($1, '+', $3); }
-	|	operand '/' operand			{ $$ = Expr::makeOpBinary($1, '/', $3); }
-	|	operand '*' operand			{ $$ = Expr::makeOpBinary($1, '*', $3); }
-	|	operand '%' operand			{ $$ = Expr::makeOpBinary($1, '%', $3); }
-	|	operand '^' operand			{ $$ = Expr::makeOpBinary($1, '^', $3); }
+	|	operand '-' operand			{ $$ = Expr::makeOpBinary($1, kOpSubtract, $3); }
+	|	operand '+' operand			{ $$ = Expr::makeOpBinary($1, kOpAdd, $3); }
+	|	operand '/' operand			{ $$ = Expr::makeOpBinary($1, kOpDivide, $3); }
+	|	operand '*' operand			{ $$ = Expr::makeOpBinary($1, kOpMultiply, $3); }
+	|	operand '%' operand			{ $$ = Expr::makeOpBinary($1, kOpModulo, $3); }
+	|	operand '^' operand			{ $$ = Expr::makeOpBinary($1, kOpPower, $3); }
```

Comment:
The operator '^' presents a problem.
In the [PostgreSQL definition](https://www.postgresql.org/docs/7.4/static/functions-math.html) this is an exponentiation operator.
In [this definition](https://www.w3schools.com/sql/sql_operators.asp) they specify the operator as bitwise XOR.

Maybe we can change the names of all these operators to reference the symbol, instead of the semantics. This allows every user to decide themselves how they implement the operator.
So: kOpPower -> kOpCaret, kOpAdd -> kOpPlus, etc.